<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplineInterpolationWidget</class>
 <widget class="QWidget" name="SplineInterpolationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_15">
     <property name="title">
      <string>Parameter</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_19">
      <item>
       <widget class="QGroupBox" name="groupBox_16">
        <property name="title">
         <string>Spline Order</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <widget class="QSpinBox" name="splineOrderSpinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>5</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_17">
        <property name="title">
         <string>Spline Levels</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QSpinBox" name="splineLevelsSpinbox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_18">
        <property name="title">
         <string>Spline Control Points</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_22">
         <item>
          <widget class="QSpinBox" name="splineControlPoints">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Reference ROIs</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <item>
          <widget class="QListWidget" name="referenceROIsListWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="add_reference_roi_button">
           <property name="text">
            <string>Add Reference ROI</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>Perform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
